examples/tpm2-extendpcr/extendpcr.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2018, Ian Haken. All rights reserved.
```

Comment:
If this is your code, then it should be Copyright Google

Reply:
will fix